src/resdk/query.py

```diff
@@ -92,6 +98,10 @@ def __init__(self, resolwe, resource, slug_field="slug"):
 
         self.logger = logging.getLogger(__name__)
 
+    def _non_string_iterable(self, item) -> bool:
+        """Return thur when item is iterable but not string."""
```

Comment:
Spelling: "thur" should be changed to True, i guess?

Also, could you explain a bit what this function is necessary?

Reply:
Before there were checks where it was expected for resources to be of instance list (`_dehydrate_resources`, `_add_filter`). I believe I used them with iterable (dictkeys...) or something like that so I changed it to a more general type.